python-modules/cis_notifications/cis_notifications/event.py

```diff
@@ -72,17 +72,20 @@ def send(self, notification):
 
         [return] Dictionary of status codes by publisher.
         """
+        # Not in-memory access token?
```

Comment:
This can still send an invalid token while performing sending a large batch? We batch up to 100 messages which can take more than the 10s grace.

Reply:
the function can take up to <max time set in serverless.yml up to 15min> so potentially we could do a "up to 15min+10" of leeway

Reply:
added